src/components/common/view/Circle.js

```diff
@@ -0,0 +1,66 @@
+// @flow
+import React from 'react'
+import { View } from 'react-native'
+import { withStyles } from '../../../lib/styles'
+import normalize from '../../../lib/utils/normalizeText'
+import Text from './Text'
+
+/**
+ * Receives a number and a unit to display
+ * @param {Props} props
+ * @param {Number} [props.number] Number to show
+ * @param {String} [props.unit] Units for the number
+ * @param {Object} [props.elementStyles] Inner elements styles
+ * @param {Object} [props.style] Outer element style
+ * @returns {React.Node}
+ */
+class Circle extends React.Component {
+  render() {
+    const { children, number, styles } = this.props
+    return (
+      <View style={styles.mainBlock}>
+        <View style={styles.circle}>
+          <Text
+            fontFamily="slab"
+            fontSize={normalize(24)}
+            // fontWeight="bold"
+            color="#ffffff"
+          >
+            {number}
+          </Text>
+        </View>
+        <View style={styles.text}>
+          <Text fontFamily="Roboto" fontSize={normalize(18)} color="#42454A">
```

Comment:
Use our generic text component in component/common/view/Text
see how it used in other places
no need for normalize in fontsize in the generic component
color should be a name from theme